### ISSUE TITLE
sencha-SlideNavigation on Sencha Touch 2.2

### ISSUE THREAD

**Issue by fservidio** [wnielson/sencha-SlideNavigation#46]
Hello everyone, I was trying to run the project with Sencha Touch 2.2 source files, but rendering is quite strange, the menu icon doesn't show up and all the elements in the slide menu collapse.
Does the project need an update?

Thank you.

**Comment by hessius** [comment]
No, it's working perfectly for me on 2.2, theming has been revamped for 2.2 though. That's probably what's messing things up.

**Comment by fservidio** [comment]
Have you made changes to the configuration files? I just downloaded the slideNavigation package, then added the "touch-2.2.0" folder and renamed it "touch". That didn't work.
By the way, no errors from the browser console.

**Comment by hessius** [comment]
You're not getting any errors as it isn't a JavaScript problem, it's likely CSS / SASS based. You need to update your app.scss for 2.2, among other things they way in which icons are included has changed totally. The official sencha blog has a good rundown of what one needs to do to update their SASS files

**Comment by fservidio** [comment]
I'll give it a try, thank you.
